cairoplot/cairoplot.py

```diff
@@ -132,7 +132,7 @@ def create_surface(self, surface, width=None, height=None):
         if isinstance(surface, cairo.Surface):
             self.surface = surface
             return
-        if not type(surface) in (str, unicode): 
+        if not type(surface) in (str):
```

Comment:
Now that a list isn't needed, use isinstance?

Reply:
Fixed in fdff3f9, upstream still used `type`.